src/toil/test/cwl/cwlTest.py

```diff
@@ -168,6 +176,42 @@ def test_biocontainers(self):
                      main_args=["--beta-use-biocontainers"],
                      out_name="output1")
 
+    @slow
+    @needs_lsf
+    @unittest.skip
```

Comment:
I'd like to eventually have these run regularly. Maybe a separate pull-request-builder on ci.commonwl.org?

Reply:
It looks like there is already a `toil-conformance` test running there.  How can we help with this from our end?